templates/beego/files/controllers/services.go

```diff
@@ -0,0 +1,139 @@
+package controllers
+
+import (
+        "fmt"
+        "database/sql"
+        _ "github.com/go-sql-driver/mysql"
```

Comment:
Does the _ import here not cause problems, as it did on Gin?

Reply:
No none of the same problems that I could see.